test/jdk/javax/swing/text/html/parser/ParserStackOverflow.java

```diff
@@ -45,7 +45,7 @@ public static void main( String[] argv ) throws Exception {
         Reader reader = new StringReader(badHtml);
         kit.read(reader, doc, 0); // StackOverflowError here
 
-        System.out.println("Succeeded! (expected StackOverflowError");
+        System.out.println("Succeeded! (no StackOverflowError");
```

Comment:
```suggestion
        System.out.println("Succeeded! (no StackOverflowError)");
```
The closing parenthesis is missing in the string.